shared/metrics/src/main/java/io/pravega/shared/metrics/StatsProviderImpl.java

```diff
@@ -97,4 +103,16 @@ public DynamicLogger createDynamicLogger() {
         init();
         return new DynamicLoggerImpl(conf, metrics, new StatsLoggerImpl(getMetrics(), "pravega"));
     }
+
+    private String[] getHostTag() {
```

Comment:
Looks like this may be used in other places, for which perhaps can be exposed as an utility function in another place.

Reply:
fixed.